### DISCUSSION TITLE
Shortcut Categories

### DISCUSSION THREAD

**Discussion by brezzy1337** [dailydotdev/daily#1162] [Feature Requests]
Hey I love this dashboard so far its very clean. I do want to be able to have an option to put different shortcuts in different categories above where the current shortcuts are. Like a category for email, development, finance, etc.

**Comment by idoshamun** [comment]
Hi @brezzy1337,

Thanks for the suggestion but this is currently out of our scope, our primary focus is delivering high quality personalized feed.
I hope you understand
